pkg/server/server.go

```diff
@@ -176,6 +177,18 @@ func NewServer(cli typedCore.CoreV1Interface, cfg Config, deps Dependencies) (*S
 		r.GET("/ready/storage", gin.WrapF(deps.Storage.Probe.ReadyHandler))
 		readyProbes = append(readyProbes, deps.Storage.Probe)
 	}
+	if deps.Backup.Enabled {
+		r.GET("/ready/backup", gin.WrapF(deps.Backup.Probe.ReadyHandler))
```

Comment:
shouldn't we add it to the liveness probes in the chart?

Reply:
It is by default part of /ready :) But now you can just ask for subhealth also